README.md

```diff
@@ -519,6 +521,7 @@ The historical release notes may helpful for general information:
 
 ### Compiler
 
+- [code/gc0initial](https://github.com/golang/go/tree/cb87526ce3531557ccf69969de4c8018956b10b5/src/c) Ken Thompson. Go0 compiler initial version. 28 Mar 2008.
```

Comment:
Did you mean this is the initial version of the Go compiler?

Reply:
I think it may be the oldest one you can find from the golang repository, not complete, not work comparing with rob pike's version (the 06/15 version). it's generate C code only.  the code refactor into the later `cmd/gc/`